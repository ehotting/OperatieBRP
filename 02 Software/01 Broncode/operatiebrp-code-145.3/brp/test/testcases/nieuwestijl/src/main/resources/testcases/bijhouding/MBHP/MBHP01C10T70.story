Meta:
@auteur                 fuman
@status                 Klaar
@usecase                BY.1.MP

Narrative:
Scenario 1 Hoofdscenario

Scenario: Persoon kan maar 1 keer betrokken zijn huwelijk zonder naamswijziging
          LT: MBHP01C10T70



Given alle personen zijn verwijderd
Given enkel initiele vulling uit bestand /LO3PL-MBHP/MBHP01C10T70-Karin.xls
Given enkel initiele vulling uit bestand /LO3PL-MBHP/MBHP01C10T70-Karel.xls
Given enkel initiele vulling uit bestand /LO3PL-MBHP/MBHP01C10T70-Tom.xls

Given Pseudo-persoon 5932312865 is vervangen door ingeschreven persoon 6827429650
Given Pseudo-persoon 7531041042 is vervangen door ingeschreven persoon 6827429650

When voer een bijhouding uit MBHP01C10T70.xml namens partij 'Gemeente BRP 1'

Then is het antwoordbericht gelijk aan /testcases/bijhouding/MBHP/expected/MBHP01C10T70.xml voor expressie //brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R

Then controleer tijdstip laatste wijziging in bijhoudingsplan voor bijgehouden personen
