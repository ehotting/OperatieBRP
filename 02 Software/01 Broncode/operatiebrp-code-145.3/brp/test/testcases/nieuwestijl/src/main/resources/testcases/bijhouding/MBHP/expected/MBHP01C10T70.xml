<?xml version="1.0" encoding="UTF-8"?><brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R xmlns:brp="http://www.bzk.nl/brp/brp0200">&#13;
    <brp:stuurgegevens>&#13;
        <brp:zendendePartij>199901</brp:zendendePartij>&#13;
        <brp:zendendeSysteem>BRP</brp:zendendeSysteem>&#13;
        <brp:referentienummer>*</brp:referentienummer>&#13;
        <brp:crossReferentienummer>88409eeb-1aa5-43fc-8614-43055123a165</brp:crossReferentienummer>&#13;
        <brp:tijdstipVerzending>*</brp:tijdstipVerzending>&#13;
    </brp:stuurgegevens>&#13;
    <brp:resultaat>&#13;
        <brp:verwerking>Geslaagd</brp:verwerking>&#13;
        <brp:bijhouding>Verwerkt</brp:bijhouding>&#13;
        <brp:hoogsteMeldingsniveau>Geen</brp:hoogsteMeldingsniveau>&#13;
    </brp:resultaat>&#13;
    <brp:voltrekkingHuwelijkInNederland brp:objecttype="AdministratieveHandeling">&#13;
        <brp:partijCode>507013</brp:partijCode>&#13;
        <brp:tijdstipRegistratie>*</brp:tijdstipRegistratie>&#13;
        <brp:bijgehoudenPersonen>*</brp:bijgehoudenPersonen>
        <brp:bijhoudingsplan brp:objecttype="Bijhoudingsplan">&#13;
            <brp:bijhoudingsvoorstelPartijCode>507013</brp:bijhoudingsvoorstelPartijCode>&#13;
            <brp:bijhoudingsplanPersonen>&#13;
                <brp:bijhoudingsplanPersoon brp:objecttype="BijhoudingsplanPersoon">&#13;
                    <brp:persoon brp:objecttype="Persoon">&#13;
                        <brp:afgeleidAdministratief>&#13;
                            <brp:tijdstipLaatsteWijziging>*</brp:tijdstipLaatsteWijziging>&#13;
                        </brp:afgeleidAdministratief>&#13;
                        <brp:identificatienummers>&#13;
                            <brp:burgerservicenummer>439522249</brp:burgerservicenummer>&#13;
                        </brp:identificatienummers>&#13;
                        <brp:bijhouding>&#13;
                            <brp:partijCode>507022</brp:partijCode>&#13;
                        </brp:bijhouding>&#13;
                    </brp:persoon>&#13;
                    <brp:situatieNaam>Automatische fiat</brp:situatieNaam>&#13;
                </brp:bijhoudingsplanPersoon>&#13;
                <brp:bijhoudingsplanPersoon brp:objecttype="BijhoudingsplanPersoon">&#13;
                    <brp:persoon brp:objecttype="Persoon">&#13;
                        <brp:afgeleidAdministratief>&#13;
                            <brp:tijdstipLaatsteWijziging>*</brp:tijdstipLaatsteWijziging>&#13;
                        </brp:afgeleidAdministratief>&#13;
                        <brp:identificatienummers>&#13;
                            <brp:burgerservicenummer>460014249</brp:burgerservicenummer>&#13;
                        </brp:identificatienummers>&#13;
                        <brp:bijhouding>&#13;
                            <brp:partijCode>507022</brp:partijCode>&#13;
                        </brp:bijhouding>&#13;
                    </brp:persoon>&#13;
                    <brp:situatieNaam>Automatische fiat</brp:situatieNaam>&#13;
                </brp:bijhoudingsplanPersoon>&#13;
            </brp:bijhoudingsplanPersonen>&#13;
        </brp:bijhoudingsplan>&#13;
    </brp:voltrekkingHuwelijkInNederland>&#13;
</brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R>
